src/storage/metrics.rs

```diff
@@ -1,34 +1,24 @@
 // Copyright 2016 TiKV Project Authors. Licensed under Apache-2.0.
 
-use prometheus::local::*;
 use prometheus::*;
 use prometheus_static_metric::*;
 
 use std::cell::RefCell;
 use std::mem;
 use std::time::Duration;
 
+use crate::server::metrics::{GcKeysCF as ServerGcKeysCF, GcKeysDetail as ServerGcKeysDetail};
 use crate::storage::kv::{FlowStatistics, FlowStatsReporter, Statistics};
 use tikv_util::collections::HashMap;
 
 struct StorageLocalMetrics {
-    local_sched_histogram_vec: LocalHistogramVec,
-    local_sched_processing_read_histogram_vec: LocalHistogramVec,
-    local_kv_command_keyread_histogram_vec: LocalHistogramVec,
-    local_kv_command_counter_vec: LocalIntCounterVec,
-    local_sched_commands_pri_counter_vec: LocalIntCounterVec,
     local_scan_details: HashMap<&'static str, Statistics>,
```

Comment:
maybe better to change it to `HashMap<CommandKind, Statistics>` to avoid the match in Line 33